src/helpers/passable_args/passable.helpers.args.spec.js

```diff
@@ -13,9 +11,9 @@ describe('Test Passable arguments logic', () => {
     });
 
     it('Should throw an exception when given a single argument of the wrong type', () => {
-        expect(passableArgs.bind(null, ['basic'])).to.throw("[Passable]: Failed to execute 'passableArgs': Unexpected string, expected function");
-        expect(passableArgs.bind(null, [{}])).to.throw("[Passable]: Failed to execute 'passableArgs': Unexpected object, expected function");
-        expect(passableArgs.bind(null, [1])).to.throw("[Passable]: Failed to execute 'passableArgs': Unexpected number, expected function");
+        expect(passableArgs.bind(null, ['basic'])).to.throw("[Passable]: Failed to execute 'passableArgs': Unexpected string, expected function.");
+        expect(passableArgs.bind(null, [{}])).to.throw("[Passable]: Failed to execute 'passableArgs': Unexpected object, expected function.");
+        expect(passableArgs.bind(null, [1])).to.throw("[Passable]: Failed to execute 'passableArgs': Unexpected number, expected function.");
```

Comment:
usually the common pattern is to put 1 `except` condition in an `it` - what do you think about splitting it?

Reply:
IMO it adds no value here as all `expects` test virtually the same thing.